### DISCUSSION TITLE
How to self host?

### DISCUSSION THREAD

**Discussion by cruizba** [pyrou/traefik.me#3] [Q&A]
I really like traefik.me for development purposes.

WebRTC requires SSL by default, and developing on a LAN using traefik.me is really useful.

I would like to deploy it on my own domain. Is there any documentation available on how to do that?

I can deploy it by myself since I see the configuration file with all the necessary parameters. I'm just checking to make sure I haven't missed any documentation.

**Comment by thepiwo** [comment]
would be super cool to have a guide for this
